### ISSUE TITLE
Comment from IBM: Redundant Entry needs to be more tightly scoped in the SC language

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#1410]
IBM recommends some minor changes to tighten the SC scope.
Although the Understanding document provides some clarity and containment on the scope of the SC, that qualification should be part of the SC language for 3.3.8, to avoid any changes that may take place in the Understanding document translation, etc.

First, the following sentence appears in the Understanding document:
>This Success Criterion pertains to the current user session, and is not applicable a user returns after closing a session or navigating away.

This point can be brought into the normative language preamble by the addition of the information about "the current user session", or can be added as a normative note:

Option 1:
> For steps in a process, information previously entered by or provided to the user that is required on subsequent steps of the same user session is either:

Option 2:
>Note: "This SC pertains to the current user session, and is not applicable when a user returns after closing a session or navigates away.

Second, the exception could clarify that in circumstances where the information is not longer relevant it should not be used. It is possible to understand this from the phrase "that is required on subsequent step"; however, an extended exception such as the following may be clearer:
>Exception: When re-entering the information is essential, or when previously entered information is no longer valid.

Closes #1335

**Comment by mbgower** [comment]
@alastc just making sure you saw this.

**Comment by mbgower** [comment]
@Helixopp have you had a chance to review this yet? Seem like pretty straight forward changes, and I'd like to get them in for survey unless there's a conflict with other changes, or a catch I'm missing?

**Comment by Helixopp** [comment]
My apologies for the extremely delayed response. I am in favor of:

Option 2:

Note: "This SC pertains to the current user session, and is not applicable when a user returns after closing a session or navigates away.
Second, the exception could clarify that in circumstances where the information is not longer relevant it should not be used. It is possible to understand this from the phrase "that is required on subsequent step"; however, an extended exception such as the following may be clearer:

Exception: When re-entering the information is essential, or when previously entered information is no longer valid.

**Comment by alastc** [comment]
Group decision from the meeting: Accept the PR, noting that the SC text will be updated by another aspect to:
> Information previously entered by or provided to the user that is required to be entered again in the same process and in the same user session is either:

**Comment by alastc** [comment]
As this was accepted, closing the issue.
